src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline.
+ */
+public class Deadline extends Task {
+    LocalDateTime time;
+
+    public Deadline(String instr, LocalDateTime time) {
+        super(instr);
+        super.type = "DEADLINE";
+        this.time = time;
+    }
+
+    /**
+     * Getter for the <code>time</code> of <code>Deadline</code>.
+     */
+    public LocalDateTime getTime() {
+        return time;
+    }
+
+    @Override
+    public String toString() {
+        if (super.done) {
```

Comment:
The variable name `done` can be changed to follow our coding guidelines (to sound boolean)